src/main/java/teamzombie/pokez/setup/Registration.java

```diff
@@ -57,6 +59,8 @@ public static void init() {
 	public static final RegistryObject<Item> Rabbit_Item = ITEMS.register("rabbit_item", () -> new PokemonAnimalItem());
 	public static final RegistryObject<Item> Sheep_Item = ITEMS.register("sheep_item", () -> new PokemonAnimalItem());
 	public static final RegistryObject<Item> Wolf_Item = ITEMS.register("wolf_item", () -> new PokemonAnimalItem());
-
+	public static final RegistryObject<Item> Pokeball = ITEMS.register("pokeball", () -> new PokeballItem());
```

Comment:
Can we seperate the pokeball registrations from the animal item registrations with a comment just for clean code purposes